libs/etcd_broker/src/subscription_key.rs

```diff
@@ -0,0 +1,310 @@
+//! Etcd broker keys, used in the project and shared between instances.
+//! The keys are split into two categories:
+//!
+//! * [`SubscriptionFullKey`] full key format: `<cluster_prefix>/<tenant>/<timeline>/<node_kind>/<operation>/<node_id>`
```

Comment:
FWIW I'm not a fan of using 'subscription' word here all around. This is just a schema, and subscribing to changes is only a (one) way to use it.